### PR TITLE
Add field 'pullRequest' as pull request title, number with link

### PR DESCRIPTION
**Pull request by nowsprinting**
Add field `pullRequest` as pull request title, number with link.

Motivation:
When triggered with pull request event, the commit is the SHA of the merge commit, and the message is "HEAD into BASE".
I can trace the pull request page from `job` or `workflow`, but it would be nice to be able to refer to the pull request directly.

Note:
When it is triggered by push event, this field shows "n/a".

~~Discussions:
Which is the best format for the pullRequest field?
Current: `<url | title #numer>`
Alternative: `title <html_url | #numer>`~~

### CODE REVIEW

**Comment by nowsprinting** [comment]
Thanks!

> I prefer `<url | title #numer>`.

It's the current format.

**Review by 8398a7 — CHANGES_REQUESTED** [review]
Sorry, I forgot to submit review.

File: docs/content/fields.md [hunk 1]
```diff
@@ -19,22 +19,23 @@ steps:
...
+| repo        | `AS_REPO`               | A working repository name                                   |
+| commit      | `AS_COMMIT`             | commit hash                                                 |
+| eventName   | `AS_EVENT_NAME`         | trigger event name                                          |
+| ref         | `AS_REF`                | git reference                                               |
+| workflow    | `AS_WORKFLOW`           | Generate a workflow link from git sha                       |
+| message     | `AS_MESSAGE`            | commit message                                              |
+| author      | `AS_AUTHOR`             | The author who pushed                                       |
+| job         | `AS_JOB`                | Generate a job run link of the job that was executed        |
+| took        | `AS_TOOK`               | Execution time for the job                                  |
+| pullRequest | `AS_PULL_REQUEST`       | Pull Request title, number with link                        |
```
[the remark this answers is not in the document]
Comment (nowsprinting) [reply docs/content/fields.md:38]: Fixed in 83b844f

**Review by 8398a7 — APPROVED** [review]

**Comment by 8398a7** [comment]
thanks!

**Pull request merged**